README.md

```diff
@@ -3,33 +3,29 @@ This is the source code for <http://tutorials.codebar.io>
 ## Getting started
 
 This is a [GitHub Pages](https://pages.github.com/) repo, so you can render the pages with [Jekyll](http://jekyllrb.com/).
+All you need is to make sure you have installed the version of Ruby indicated in
```

Comment:
Ah sorry! Of course, I am happy to fix it :smiley: 
Happy to take your suggestions as well, if you have any!